components/org.wso2.carbon.status.dashboard.web/src/overview/WorkerOverview.jsx

```diff
@@ -400,27 +402,30 @@ export default class WorkerOverview extends React.Component {
         } else if (this.state.isApiCalled && ((WorkerOverview.hasNodes(this.state.clustersList)) &&
                 (WorkerOverview.hasNodes(this.state.managerClusterList)))) {
             return (
+
                 <div style={styles.background}>
                     <div style={{height: 20, padding: 20, backgroundColor: '#222222'}}>
                         {this.renderAddWorkerFlotting()}
-                        <div className="toggle">
-                            <Toggle labelPosition="left"
-                                    label={<b>Auto Sync</b>}
-                                    labelStyle={{color: '#dedede', fontSize: 18}}
-                                    toggled={this.state.enableAutoSync}
-                                    onToggle={this.autoSync}
-                                    thumbStyle={{backgroundColor: 'grey'}}
-                                    thumbSwitchedStyle={{backgroundColor: '#f17b31'}}
-                                    trackSwitchedStyle={{backgroundColor: '#f17b31'}}>
-                            </Toggle>
+                        <div className="toggle" style={{marginTop: '-2%', marginRight: '16%'}}>
+                            <Button style={styles.navBtn} onClick={() => {
+                                this.autoSync();
+                            }}>
+                                <Sync color={this.state.btnColor}/>
```

Comment:
Rather than color change, Shall we change the icons? For sync disabled: https://material.io/tools/icons/?icon=sync_disabled&style=outline. I believe we just have to add a flow based on the value of enableAutoSync in the state as follows,
`
<Button>
{ this.state.enableAutoSync == {true} && 
<Sync>
.
.
}
{
 this.state.enableAutoSync == {true} && 
<SyncDisable>
..
}
</Button>
`

Reply:
Yes, change the icon and color when toggle.